### PR TITLE
add sdn and ssi type

### LINKED ISSUE #313: client: add enums for SDN type
SDN types should include:
- individual
- entity
- aircraft
- vessel

_Originally posted by @vxio in https://github.com/moovfinancial/identity/pull/72#r516396877_

### PR DESCRIPTION
**Pull request by vxio**
Fixes #313

### CODE REVIEW

File: api/client.yaml [hunk 1]
```diff
@@ -996,6 +994,14 @@ components:
         match:
           type: number
           example: 0.91
+    OfacType:
```
Comment (adamdecaf) [inline api/client.yaml:997]: This should probably be called `SdnType`

Comment (vxio) [reply api/client.yaml:997]: I originally had it as `SdnType`, but noticed `SSI` uses the same type
https://github.com/moov-io/watchman/blob/a0df66ec303b421fe5e488e00c165320c3182a08/client/api/openapi.yaml#L1442

Comment (adamdecaf) [reply api/client.yaml:997]: I don't believe the SSI list contains vessels right now. It's a different list, so I'd have `SdnType` and `SsiType`. 

https://home.treasury.gov/policy-issues/financial-sanctions/consolidated-sanctions-list/sectoral-sanctions-identifications-ssi-list

Comment (vxio) [reply api/client.yaml:997]: good point, I'll do that

File: api/client.yaml [hunk 2]
```diff
@@ -996,6 +994,20 @@ components:
...
+        - individual
+        - entity
+        - vessel
+        - aircraft
+    SsiType:
+      description: 'Used for classifying SSIs — typically represents an individual or company'
+      type: string
+      enum:
+        - individual
+        - entity
```
[the remark this answers is not in the document]
Comment (adamdecaf) [reply api/client.yaml:1010]: Nope. Within OFAC/SDNs anything is an entity. Companies are a sub-type of an entity. 

Individuals are categorized by the third column in the CSV:
```
4273,"AMEZQUITA MENESES, Salustio","individual","SDNT",-0- ,-0- ,-0- ,-0- ,-0- ,-0- ,-0- ,"DOB 01 Jul 1946; Cedula No. 14943885 (Colombia)."
```

Whereas companies are categorized by the null value (`-0-`) in the CSV:
```
4268,"AGROPECUARIA Y REFORESTADORA HERREBE LTDA.",-0- ,"SDNT",-0- ,-0- ,-0- ,-0- ,-0- ,-0- ,-0- ,-0- 
```

**Review by adamdecaf — APPROVED** [review]

File: api/client.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vxio) [reply api/client.yaml:1010]: Ahh thanks for the info; I wonder if we'll ever have to distinguish subtypes.

**Pull request merged**